jobs/build/pre-release/Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def prerelease = load("pipeline-scripts/release.groovy")
+    def buildlib = prerelease.buildlib
+    def commonlib = prerelease.commonlib
+    def quay_url = "quay.io/openshift-release-dev/ocp-release-nightly"
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'FROM_RELEASE_TAG',
+                        description: 'Build tag to pull from (i.e. 4.1.0-0.nightly-2019-04-22-005054), pre-release job the release name is same as FROM_RELEASE_TAG',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'DRY_RUN',
+                        description: 'Only do dry run test and exit.',
+                        $class: 'BooleanParameterDefinition',
+                        defaultValue: false
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                    commonlib.suppressEmailParam(),
+                ]
+            ],
+            disableConcurrentBuilds()
+        ]
+    )
+
+    commonlib.checkMock()
+
+    buildlib.cleanWorkdir("${env.WORKSPACE}")
+
+    try {
+        sshagent(['aos-cd-test']) {
+            release_info = ""
+            from_release_tag = "${params.FROM_RELEASE_TAG}"
+            name = from_release_tag.split("nightly")[0]+"nightly"
+            // must be able to access remote registry for verification
+            buildlib.registry_quay_dev_login()
+
+            stage("versions") { prerelease.stageVersions() }
+
+            stage("validation") { prerelease.stageValidation(quay_url, from_release_tag, "") }
+
+            stage("payload") { prerelease.stageGenPayload(quay_url, from_release_tag, from_release_tag, "", "", "") }
+
+            stage("tag prerelease") { prerelease.stageTagRelease(quay_url, from_release_tag) }
```

Comment:
I think this should be skipped in pre-release. AIUI (@tbielawa help me out here) this was used in `release` job because we actually gave the release in quay a different name, and needed to tag that back to the `release` repo in CI. But here, we're just reusing the name, not creating anything new. Doing this tag would actually overwrite the original release image with its copy.

Reply:
:+1: